security/advancedtls/advancedtls_test.go

```diff
@@ -192,17 +189,37 @@ func TestClientServerHandshake(t *testing.T) {
 			nil,
 			false,
 		},
+		// Client: set clientGetRoot and clientVerifyFunc, and bad custom verification function
```

Comment:
Is this setting `clientVerifyFunc` to `bad custom verification function`, not setting `clientVerifyFunc` and `bad custom verification function`?